modules/claims_api/app/services/claims_api/evidence_waiver_document_service.rb

```diff
@@ -25,11 +26,23 @@ def generate_body(claim:, doc_type:, pdf_path:, ptcpnt_vet_id:)
       auth_headers = claim.auth_headers
       veteran_name = compact_veteran_name(auth_headers['va_eauth_firstName'],
                                           auth_headers['va_eauth_lastName'])
-      file_name = build_file_name(veteran_name:, identifier: claim.claim_id, suffix: '5103')
+      file_name = built_file_name(veteran_name:, claim:, suffix: FORM_SUFFIX)
       tracked_item_ids = claim.tracked_items&.map(&:to_i) if claim&.has_attribute?(:tracked_items)
 
       generate_upload_body(claim_id: claim.claim_id, system_name: 'VA.gov', doc_type:, pdf_path:, file_name:,
                            birls_file_number: nil, participant_id: ptcpnt_vet_id, tracked_item_ids:)
     end
+
+    def built_file_name(veteran_name:, claim:, suffix:)
+      if dependent_filing?(claim)
+        build_dependent_file_name(veteran_name:, identifier: claim.claim_id, suffix:)
+      else
+        build_file_name(veteran_name:, identifier: claim.claim_id, suffix:)
+      end
```

Comment:
I was looking for a way to include a flexible way of providing this from the parent class since this prefix will likely be utilized in other spots _I suspect_.  But open to better approaches to handling that.

Reply:
If we are likely to use the prefix in other forms, wouldnʼt it make sense to move this logic to the parent/base class?

Reply:
@tycol Let me know what you think of the approach in 005de23.  As you will see, I just switched to passing the dependent marker into the existing file name builder and the stole you prefix from above code to use there.

Reply:
Looks great!